src/main.rs

```diff
@@ -166,6 +219,33 @@ fn main() -> Result<()> {
             .await
             {
                 Ok(fetched_policy) => {
+                    if verify_enabled {
+                        if verified_manifest_digest.is_none() {
+                            // when deserializing keys we check that have keys to
+                            // verify. We will always have a digest manifest
+                            fatal_error("Trying to verify but no keys were passed".to_string());
```

Comment:
I would change the error message to be more generic, this will make it work also for keyless signing. I know it's a bit sill, because this message is never shown (we raise an error way earlier than that). However...

Maybe use something like `Verification of the policy failed`

Reply:
Done in https://github.com/kubewarden/policy-server/pull/132/commits/8c81056fa166d5d9140f0f51b7d403284da5354e!